etcdctl/ctlv3/command/ep_command.go

```diff
@@ -126,7 +126,7 @@ func epHealthCommandFunc(cmd *cobra.Command, args []string) {
 			// get a random key. As long as we can get the response without an error, the
 			// endpoint is health.
 			ctx, cancel := commandCtx(cmd)
-			_, err = cli.Get(ctx, "health")
+			_, err = cli.Get(ctx, "health", v3.WithSerializable())
```

Comment:
It would be safe not to add `v3.WithSerializable()` as a parameter, because it changes the semantics of get e.g. an isolated member of a cluster can return the result without errors.
How about adding a new flag to the command like `--consistency="s"` of get command?

Reply:
I think you are right, but add a new flag to the command like `--consistency="s"`  when I exec `endpoint health` command, it maybe not flexible for `endpoint` operation with `get` option. 
I want it to be automatically recognized in the etcd server, if the endpoint is learner to do health check.
I haven't figured out a better way yet for learner's health check. have a good idea? ^_^

Reply:
I see, let me consider that 🤔  But I feel adding the new flag would be the simplest way.

Reply:
may be you are right, let me think again. thanks for your suggestion